packages/cli/src/ui/components/InputPrompt.test.tsx

```diff
@@ -1495,7 +1495,7 @@ describe('InputPrompt', () => {
 
       const outputLines = frame!.split('\n');
       // The number of lines should be 2 for the border plus 3 for the content.
-      expect(outputLines.length).toBe(5);
+      expect(outputLines.length).toBe(6);
```

Comment:
as far as I can tell just Ink being weird now that there is a fixed width. reality is there should have always shown up as a line but Ink wasn't including one in the output. *shrug*